components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/FieldValidationConfigurationHandler.java

```diff
@@ -50,4 +51,14 @@ public interface FieldValidationConfigurationHandler {
      */
     boolean validateValidationConfiguration(List<RulesConfiguration> configurationList)
             throws InputValidationMgtClientException;
+
+    /**
+     * Perform post actions after validation configuration are updated.
+     *
+     * @param tenantDomain      Tenant Domain.
+     * @param configuration     Updated validation configuration.
+     */
+    default void postValidationConfigurationUpdateHandler(String tenantDomain, ValidationConfiguration configuration)
```

Comment:
Handler is a noun... let's rename the method name as a verb.
```suggestion
    default void handlePostValidationConfigurationUpdate(String tenantDomain, ValidationConfiguration configuration)
```

Reply:
Addressed

Reply:
This is not addressed

Reply:
Addressed